octobot_trading/personal_data/orders/order_util.py

```diff
@@ -435,3 +437,68 @@ async def get_order_size_portfolio_percent(exchange_manager, order_amount, side,
 
 def generate_order_id():
     return str(uuid.uuid4())
+
+
+async def apply_order_storage_details_if_any(order, exchange_manager, pending_groups):
+    # only real orders can be updated by stored orders
+    if not exchange_manager.storage_manager.orders_storage \
+            or not exchange_manager.storage_manager.orders_storage.should_store_date():
+        return
+    order_details = await exchange_manager.storage_manager.orders_storage.get_startup_order_details(order.order_id)
+    if order_details:
+        order.update_from_storage_order_details(order_details)
+        await create_orders_storage_related_elements(order, order_details, exchange_manager, pending_groups)
+
+
+async def create_orders_storage_related_elements(order, order_storage_details, exchange_manager, pending_groups):
```

Comment:
I think we should also put it in order factory

Reply:
I didn't put it there because it's also creating groups, but we could also put it there. `_create_storage_self_managed_orders_from_group` could also be but there as this one it is creating orders